.github/workflows/tests_primary.yml

```diff
@@ -153,3 +153,40 @@ jobs:
       if: always() && matrix.os != 'ubuntu-latest'
     - run: xvfb-run npm run test-web
       if: always() && matrix.os == 'ubuntu-latest'
+
+  test-package-installations:
+    name: "Installation Test: ${{ matrix.os }}, Node ${{ matrix.node_version }}"
+    runs-on: ${{ matrix.os  }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+        - ubuntu-latest
+        - macos-latest
+        - windows-latest
+        node_version:
+        - "^18.0.0"
+        include:
+        - node: "12.0.0"
+          os: ubuntu-latest
+        - node: "^12.0.0"
+          os: ubuntu-latest
+        - node: "^14.1.0"  # pre 14.1, zip extraction was broken (https://github.com/microsoft/playwright/issues/1988)
+          os: ubuntu-latest
+        - node: "^16.0.0"
+          os: ubuntu-latest
```

Comment:
Turns out this is not how this syntax works. We'll instead want to use the full matrix, and use exclude.